![me](/images/zl.png) *Hello / Guten Tag*

I am a molecular and computational biologist broadly interested in gene expression and regulation, functional genomics, and drug targets discovery. I explore `multi-omics` data, perform `bioinformatics` analysis, and apply machine learning `modelling`.

I work mostly on parasitic flatworms of *Schistosoma* species.

![mywordcloud](/images/wordcloud2.png)

## Professional experience

- PhD @Giessen (*Dr.rer.nat* in Biology) work on isolation, molecular characterisation, and comparative transcriptomics of *S. mansoni* reproductive organs. 
- PostDoc @Giessen: single cell preparation and sequencing (scRNA-seq); data integration and visualisation
- [Computational Biologist](https://www.sanger.ac.uk/person/lu-zhigang/) @Sanger: genome assembly and [annotation](https://zglu.github.io/html_slides_charts/v7-annotation.html#/), omics data analysis and integration, bioinformatics, machine learning modelling, data portal development
- Data Scientist/Bioinformatician: project conception, bioinformatics analysis, scRNA-seq data analysis   

## Selected work 

- ### Time-series scRNA-seq (ongoing)

> Aims to track the dynmics in cell population and gene expression, and potential interactions.

![scrna](/images/sc1.png)

- ### Prediction of analytical conditions based on machine learning models (ongoing)

> Supervised learning on pre-experimental data, model optimisation, and prediction on optimal conditions. Model was supported by validation experiment.

![exp-pred](/images/desp.png)

- ### Multi-omics data integration and modelling (ongoing)
> Combined RNA-seq and ChIP-seq data were used for [TSS determination](https://zglu.github.io/html_slides_charts/tss_isoseq_chipseq.html) and [modelling](https://zglu.github.io/html_slides_charts/modelling-chipseq-rnaseq.html#/)

![rna-chip](/images/rna-chip.png)

- ### Multi-scale data integration
> RNA-seq data from various studies were [combined and re-analysed](https://zglu.github.io/html_slides_charts/rnaseq-expression.html#slide=1), prio knowledge added and information shared. -- [Schisto.xyz](https://schisto.xyz)

![schisto.xyz](/images/logo-v7.jpg)

## Selected publications

- Rawlinson, K. A., Reid, A. J., **Lu, Z.**, Driguez, P., Wawer, A., Coghlan, A., et al. (2021). Daily rhythms in gene expression of the human parasite *Schistosoma mansoni*. [BMC Biol.](https://bmcbiol.biomedcentral.com/articles/10.1186/s12915-021-01189-9) 19, 255. doi:10.1186/s12915-021-01189-9.
> contributed to bioinformatics alalysis and visualisation

- Wang, J., Paz, C., Padalino, G., Coghlan, A., **Lu, Z.**, Gradinaru, I., et al. (2020). Large-scale RNAi screening uncovers therapeutic targets in the parasite *Schistosoma mansoni*. [Science](https://www.science.org/doi/10.1126/science.abb7699) 369. doi:10.1126/science.abb7699.
> contributed to bioinformatics analysis

- Diaz Soria, C. L., Lee, J., Chong, T., Coghlan, A., Tracey, A., Young, M. D., ... **Lu Z.** et al. (2020). Single-cell atlas of the first intra-mammalian developmental stage of the human parasite *Schistosoma mansoni*. [Nat. Commun.](https://www.nature.com/articles/s41467-020-20092-5) 11, 6411. doi:10.1038/s41467-020-20092-5.
>contributed to bioinformatics analysis

- **Lu, Z.**, and Berriman, M. (2018). Meta-analysis of RNA-seq studies reveals genes responsible for life stage-dominant functions in *Schistosoma mansoni*. [bioRxiv](https://www.biorxiv.org/content/10.1101/308189v1.full), 308189.

![](https://www.biorxiv.org/content/biorxiv/early/2018/04/25/308189/F2.medium.gif)

- **Lu, Z.**, Sessler, F., Holroyd, N., Hahnel, S., Quack, T., Berriman, M., et al. (2016). Schistosome sex matters: A deep view into gonad-specific and pairing-dependent transcriptomes reveals a complex gender interplay. [Sci. Rep.](https://www.nature.com/articles/srep31150) 6. doi:10.1038/srep31150.
>featured in [Science](https://www.science.org/doi/10.1126/science.2016.353.6304.twil) Editor’s Choice “Sexual development in schistosomes”

![in Science](/images/in-science.png)

## Bioinformatics and coding

I write code mostly in <mark>Bash, R and Python</mark>.

- ### Testing functional enrichment in genes of interest
>Can use GO, Pfam, InterPro, or KEGG Pathway annotations [Scripts](https://github.com/zglu/Gene-function-enrichment)

![go](/images/go-bar.png)

- ### Identify chromosomal clusters of genes with similar functions
>based on protein domain or gene family information. [Scripts](https://github.com/zglu/FunctionalClusters_adjacentGenes) 

![cluster](/images/clusters.png)

- ### Python-based scripts to manipulate GFF files
>gff2fasta, extract up-downstream sequences, etc. [Scripts](https://github.com/zglu/Scripts_Bioinfo/tree/master/Python)

- ### Interactive charting (fun project)
>Javascript-based tool for interactive plotting: [ivis](ivis.netlify.app)

![ivis](/images/ivis.png)

---
