---
title: "SingleCellAtlas"
menu: "main"
draft: false
---

## Single Cell Atlas

Collection of single cell atlas papers and data portals.

### [Mapping cell types across human tissues](https://www.science.org/doi/10.1126/science.abq2116)

- [Single-nucleus cross-tissue molecular reference maps toward understanding disease gene function](https://www.science.org/doi/10.1126/science.abl4290) [https://singlecell.broadinstitute.org/single_cell/study/SCP1479]
- [The Tabula Sapiens: A multiple-organ, single-cell transcriptomic atlas of humans](https://www.science.org/doi/10.1126/science.abl4896) [https://tabula-sapiens-portal.ds.czbiohub.org/]
- [Cross-tissue immune cell analysis reveals tissue-specific features in humans](https://www.science.org/doi/10.1126/science.abl5197) [https://www.celltypist.org/]
- [Mapping the developing human immune system across organs](https://www.science.org/doi/10.1126/science.abo0510)


***

### [Spatio-temporal development of model organisms](https://www.cell.com/consortium/spatiotemporal-omics)

- [Spatiotemporal transcriptomic atlas of mouse organogenesis using DNA nanoball-patterned arrays](https://www.cell.com/cell/fulltext/S0092-8674(22)00399-3) [https://db.cngb.org/stomics/mosta/]
- [High-resolution 3D spatiotemporal transcriptomic maps of developing Drosophila embryos and larvae](https://www.cell.com/developmental-cell/fulltext/S1534-5807(22)00246-5) [https://db.cngb.org/stomics/flysta3d/]
- [Spatiotemporal mapping of gene expression landscapes and developmental trajectories during zebrafish embryogenesis](https://www.cell.com/developmental-cell/fulltext/S1534-5807(22)00249-0) [https://db.cngb.org/stomics/zesta/]
- [The single-cell stereo-seq reveals region-specific cell subtypes and transcriptome profiling in Arabidopsis leaves](https://www.cell.com/developmental-cell/fulltext/S1534-5807(22)00251-9) [https://db.cngb.org/stomics/datasets/STDS0000104]


***

### Integration of single cell data

#### Benchmarking analysis

- [Benchmarking spatial and single-cell transcriptomics integration methods for transcript distribution prediction and cell type deconvolution](https://www.nature.com/articles/s41592-022-01480-9)

#### Selected tools
- PASTE: [Alignment and integration of spatial transcriptomics data](https://www.nature.com/articles/s41592-022-01459-6) ([Python](https://github.com/raphael-group/paste))
- GLUE: [Multi-omics single-cell data integration and regulatory inference with graph-linked embedding](https://www.nature.com/articles/s41587-022-01284-4) ([Python/R](https://github.com/gao-lab/GLUE))
- CellTrek: [Spatial charting of single-cell transcriptomes in tissues](https://www.nature.com/articles/s41587-022-01233-1) ([R](https://github.com/navinlabcode/CellTrek))
- [cell2location](https://cell2location.readthedocs.io/en/latest/): [Cell2location maps fine-grained cell types in spatial transcriptomics](https://www.nature.com/articles/s41587-021-01139-4) ([Python](https://github.com/BayraktarLab/cell2location/))

### Single cell data interactive visualisation

- [Comparison of visualization tools for single-cell RNAseq data](https://academic.oup.com/nargab/article/2/3/lqaa052/5877814)
- [Seurat interactive plotting](https://satijalab.org/seurat/articles/visualization_vignette.html#interactive-plotting-features-1) (`HoverLocator()`)

### Other tools

- sceasy: easy conversion of different single-cell data formats ([R](https://github.com/cellgeni/sceasy))
