<!DOCTYPE html>
<html lang="en-US">

<head>
  <meta http-equiv="X-Clacks-Overhead" content="GNU Terry Pratchett" />
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
<link rel="shortcut icon" href="https://zglu.github.io/images/favicon.png" />
<title>SingleCellTools | Zhigang Lu</title>
<meta name="title" content="SingleCellTools" />
<meta name="description" content="Selected tools for single-cell and spatial transcriptomics.
scRNA-seq  ALRA: Zero-preserving imputation SingelR: Reference-based cell type annotation HotSpot: identifies informative gene modules (pip install hotspotsc==0.9.1) SCENIC/pySCENIC: Single-cell regulatory network and clustering MACA: marker-based automatic cell-type annotation / ScTypeDB: cell-type marker database SpaOTsc: spatial optimal transport for SC data(py)   spatial transcriptomics  Giotto: toolbox for integrative analysis of spatial data SSAM (lite): segmentation-free inference of cell types SpatialDE: identification of spatially variable genes (py) Spark: Statistical analysis of spatial expression patterns BayesSpace: Clustering and Resolution Enhancement of Spatial Transcriptomes   Integration of single cell data Benchmarking analysis  Benchmarking spatial and single-cell transcriptomics integration methods for transcript distribution prediction and cell type deconvolution  Selected tools  PASTE: Alignment and integration of spatial transcriptomics data from multiple adjacent tissue slices (Python) GLUE: Multi-omics single-cell data integration and regulatory inference with graph-linked embedding (Python/R) CellTrek: Spatial charting of single-cell transcriptomes in tissues (R) Tangram: aligns sc/snRNA-seq data to spatial data (Python) SPOTlight: integration of Spatial Transcriptomics with scRNA-seq data to infer the location of cell types and states (R) Harmony: Fast, sensitive and accurate integration of single-cell data (R) cell2location: Cell2location maps fine-grained cell types in spatial transcriptomics (Python)   Single cell data interactive visualisation  Comparison of visualization tools for single-cell RNAseq data Seurat interactive plotting (HoverLocator())  Automated cell type annotation Turorial Reference-based  RCTD/spacexr (R) scmap SingleR CelliD: clustering-free extraction of per-cell gene signatures (R)  Marker-based  AUCell: cells with active gene sets GSEA/GSVA (for clusters) SCINA (for cells)  Knowledge-based clustering and annotation  UNIFAN: using known gene sets &#43; neural network   Other tools  sceasy: easy conversion of different single-cell data formats (R)  " />
<meta name="keywords" content="" />


<meta property="og:title" content="SingleCellTools" />
<meta property="og:description" content="Selected tools for single-cell and spatial transcriptomics.
scRNA-seq  ALRA: Zero-preserving imputation SingelR: Reference-based cell type annotation HotSpot: identifies informative gene modules (pip install hotspotsc==0.9.1) SCENIC/pySCENIC: Single-cell regulatory network and clustering MACA: marker-based automatic cell-type annotation / ScTypeDB: cell-type marker database SpaOTsc: spatial optimal transport for SC data(py)   spatial transcriptomics  Giotto: toolbox for integrative analysis of spatial data SSAM (lite): segmentation-free inference of cell types SpatialDE: identification of spatially variable genes (py) Spark: Statistical analysis of spatial expression patterns BayesSpace: Clustering and Resolution Enhancement of Spatial Transcriptomes   Integration of single cell data Benchmarking analysis  Benchmarking spatial and single-cell transcriptomics integration methods for transcript distribution prediction and cell type deconvolution  Selected tools  PASTE: Alignment and integration of spatial transcriptomics data from multiple adjacent tissue slices (Python) GLUE: Multi-omics single-cell data integration and regulatory inference with graph-linked embedding (Python/R) CellTrek: Spatial charting of single-cell transcriptomes in tissues (R) Tangram: aligns sc/snRNA-seq data to spatial data (Python) SPOTlight: integration of Spatial Transcriptomics with scRNA-seq data to infer the location of cell types and states (R) Harmony: Fast, sensitive and accurate integration of single-cell data (R) cell2location: Cell2location maps fine-grained cell types in spatial transcriptomics (Python)   Single cell data interactive visualisation  Comparison of visualization tools for single-cell RNAseq data Seurat interactive plotting (HoverLocator())  Automated cell type annotation Turorial Reference-based  RCTD/spacexr (R) scmap SingleR CelliD: clustering-free extraction of per-cell gene signatures (R)  Marker-based  AUCell: cells with active gene sets GSEA/GSVA (for clusters) SCINA (for cells)  Knowledge-based clustering and annotation  UNIFAN: using known gene sets &#43; neural network   Other tools  sceasy: easy conversion of different single-cell data formats (R)  " />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://zglu.github.io/singlecelltools/" /><meta property="og:image" content="https://zglu.github.io/images/share.png"/><meta property="article:section" content="" />

<meta property="og:site_name" content="Zhigang Lu" />




<meta name="twitter:card" content="summary_large_image"/>
<meta name="twitter:image" content="https://zglu.github.io/images/share.png"/>

<meta name="twitter:title" content="SingleCellTools"/>
<meta name="twitter:description" content="Selected tools for single-cell and spatial transcriptomics.
scRNA-seq  ALRA: Zero-preserving imputation SingelR: Reference-based cell type annotation HotSpot: identifies informative gene modules (pip install hotspotsc==0.9.1) SCENIC/pySCENIC: Single-cell regulatory network and clustering MACA: marker-based automatic cell-type annotation / ScTypeDB: cell-type marker database SpaOTsc: spatial optimal transport for SC data(py)   spatial transcriptomics  Giotto: toolbox for integrative analysis of spatial data SSAM (lite): segmentation-free inference of cell types SpatialDE: identification of spatially variable genes (py) Spark: Statistical analysis of spatial expression patterns BayesSpace: Clustering and Resolution Enhancement of Spatial Transcriptomes   Integration of single cell data Benchmarking analysis  Benchmarking spatial and single-cell transcriptomics integration methods for transcript distribution prediction and cell type deconvolution  Selected tools  PASTE: Alignment and integration of spatial transcriptomics data from multiple adjacent tissue slices (Python) GLUE: Multi-omics single-cell data integration and regulatory inference with graph-linked embedding (Python/R) CellTrek: Spatial charting of single-cell transcriptomes in tissues (R) Tangram: aligns sc/snRNA-seq data to spatial data (Python) SPOTlight: integration of Spatial Transcriptomics with scRNA-seq data to infer the location of cell types and states (R) Harmony: Fast, sensitive and accurate integration of single-cell data (R) cell2location: Cell2location maps fine-grained cell types in spatial transcriptomics (Python)   Single cell data interactive visualisation  Comparison of visualization tools for single-cell RNAseq data Seurat interactive plotting (HoverLocator())  Automated cell type annotation Turorial Reference-based  RCTD/spacexr (R) scmap SingleR CelliD: clustering-free extraction of per-cell gene signatures (R)  Marker-based  AUCell: cells with active gene sets GSEA/GSVA (for clusters) SCINA (for cells)  Knowledge-based clustering and annotation  UNIFAN: using known gene sets &#43; neural network   Other tools  sceasy: easy conversion of different single-cell data formats (R)  "/>



<meta itemprop="name" content="SingleCellTools">
<meta itemprop="description" content="Selected tools for single-cell and spatial transcriptomics.
scRNA-seq  ALRA: Zero-preserving imputation SingelR: Reference-based cell type annotation HotSpot: identifies informative gene modules (pip install hotspotsc==0.9.1) SCENIC/pySCENIC: Single-cell regulatory network and clustering MACA: marker-based automatic cell-type annotation / ScTypeDB: cell-type marker database SpaOTsc: spatial optimal transport for SC data(py)   spatial transcriptomics  Giotto: toolbox for integrative analysis of spatial data SSAM (lite): segmentation-free inference of cell types SpatialDE: identification of spatially variable genes (py) Spark: Statistical analysis of spatial expression patterns BayesSpace: Clustering and Resolution Enhancement of Spatial Transcriptomes   Integration of single cell data Benchmarking analysis  Benchmarking spatial and single-cell transcriptomics integration methods for transcript distribution prediction and cell type deconvolution  Selected tools  PASTE: Alignment and integration of spatial transcriptomics data from multiple adjacent tissue slices (Python) GLUE: Multi-omics single-cell data integration and regulatory inference with graph-linked embedding (Python/R) CellTrek: Spatial charting of single-cell transcriptomes in tissues (R) Tangram: aligns sc/snRNA-seq data to spatial data (Python) SPOTlight: integration of Spatial Transcriptomics with scRNA-seq data to infer the location of cell types and states (R) Harmony: Fast, sensitive and accurate integration of single-cell data (R) cell2location: Cell2location maps fine-grained cell types in spatial transcriptomics (Python)   Single cell data interactive visualisation  Comparison of visualization tools for single-cell RNAseq data Seurat interactive plotting (HoverLocator())  Automated cell type annotation Turorial Reference-based  RCTD/spacexr (R) scmap SingleR CelliD: clustering-free extraction of per-cell gene signatures (R)  Marker-based  AUCell: cells with active gene sets GSEA/GSVA (for clusters) SCINA (for cells)  Knowledge-based clustering and annotation  UNIFAN: using known gene sets &#43; neural network   Other tools  sceasy: easy conversion of different single-cell data formats (R)  ">

<meta itemprop="wordCount" content="262"><meta itemprop="image" content="https://zglu.github.io/images/share.png"/>
<meta itemprop="keywords" content="" />
<meta name="referrer" content="no-referrer-when-downgrade" />

  <style>
  body {
    font-family: Trebuchet MS, sans-serif;
    margin: auto;
    padding: 20px;
    max-width: 720px;
    text-align: left;
    background-color: #fff;
    word-wrap: break-word;
    overflow-wrap: break-word;
    line-height: 1.5;
    color: #444;
  }

  h1,
  h2,
  h3,
  h4,
  h5,
  h6,
  strong,
  b {
    color: #222;
  }

  a {
    color: #3273dc;
     
  }

  .title {
    text-decoration: none;
    border: 0;
  }

  .title span {
    font-weight: 400;
  }

  nav a {
    margin-right: 10px;
  }

  textarea {
    width: 100%;
    font-size: 16px;
  }

  input {
    font-size: 16px;
  }

  content {
    line-height: 1.6;
  }

  table {
    width: 100%;
  }

  img {
    max-width: 100%;
  }

  code {
    padding: 2px 5px;
    background-color: #f2f2f2;
  }

  pre code {
    color: #222;
    display: block;
    padding: 20px;
    white-space: pre-wrap;
    font-size: 14px;
  }

  div.highlight pre {
    background-color: initial;
    color: initial;
  }

  div.highlight code {
    background-color: unset;
    color: unset;
  }

  blockquote {
    border-left: 1px solid #999;
    color: #222;
    padding-left: 20px;
    font-style: italic;
  }

  footer {
    padding: 25px;
    text-align: center;
  }

  .helptext {
    color: #777;
    font-size: small;
  }

  .errorlist {
    color: #eba613;
    font-size: small;
  }

   
  ul.blog-posts {
    list-style-type: none;
    padding: unset;
  }

  ul.blog-posts li {
    display: flex;
  }

  ul.blog-posts li span {
    flex: 0 0 130px;
  }

  ul.blog-posts li a:visited {
    color: #8b6fcb;
  }

  @media (prefers-color-scheme: dark) {
    body {
      background-color: #333;
      color: #ddd;
    }

    h1,
    h2,
    h3,
    h4,
    h5,
    h6,
    strong,
    b {
      color: #eee;
    }

    a {
      color: #8cc2dd;
    }

    code {
      background-color: #777;
    }

    pre code {
      color: #ddd;
    }

    blockquote {
      color: #ccc;
    }

    textarea,
    input {
      background-color: #252525;
      color: #ddd;
    }

    .helptext {
      color: #aaa;
    }
  }

</style>

</head>

<body>
  <header><a href="/" class="title">
  <h2>Zhigang Lu</h2>
</a>
<nav><a href="/">Home</a>

<a href="/publications/">Publications</a>

<a href="/singlecellatlas/">SingleCellAtlas</a>

<a href="/singlecelltools/">SingleCellTools</a>


</nav>
</header>
  <main>

<content>
  <p>Selected tools for single-cell and spatial transcriptomics.</p>
<h3 id="scrna-seq">scRNA-seq</h3>
<ul>
<li><a href="https://www.nature.com/articles/s41467-021-27729-z">ALRA: Zero-preserving imputation</a></li>
<li><a href="http://bioconductor.org/books/release/SingleRBook/introduction.html">SingelR: Reference-based cell type annotation</a></li>
<li><a href="https://hotspot.readthedocs.io/en/latest/">HotSpot: identifies informative gene modules</a> (<a href="https://yoseflab.github.io/Hotspot/">pip install hotspotsc==0.9.1</a>)</li>
<li><a href="https://scenic.aertslab.org/">SCENIC/pySCENIC: Single-cell regulatory network and clustering</a></li>
<li><a href="https://github.com/ImXman/MACA">MACA: marker-based automatic cell-type annotation</a> / <a href="https://www.nature.com/articles/s41467-022-28803-w#Abs1">ScTypeDB: cell-type marker database</a></li>
<li><a href="https://www.nature.com/articles/s41467-020-15968-5">SpaOTsc: spatial optimal transport for SC data</a>(<a href="https://github.com/zcang/SpaOTsc">py</a>)</li>
</ul>
<hr>
<h3 id="spatial-transcriptomics">spatial transcriptomics</h3>
<ul>
<li><a href="https://rubd.github.io/Giotto_site/">Giotto: toolbox for integrative analysis of spatial data</a></li>
<li><a href="https://www.nature.com/articles/s41467-021-23807-4">SSAM (lite): segmentation-free inference of cell types</a></li>
<li><a href="https://github.com/Teichlab/SpatialDE">SpatialDE: identification of spatially variable genes</a> (py)</li>
<li><a href="https://github.com/xzhoulab/SPARK">Spark: Statistical analysis of spatial expression patterns</a></li>
<li><a href="http://www.bioconductor.org/packages/release/bioc/vignettes/BayesSpace/inst/doc/BayesSpace.html">BayesSpace: Clustering and Resolution Enhancement of Spatial Transcriptomes</a></li>
</ul>
<hr>
<h3 id="integration-of-single-cell-data">Integration of single cell data</h3>
<h4 id="benchmarking-analysis">Benchmarking analysis</h4>
<ul>
<li><a href="https://www.nature.com/articles/s41592-022-01480-9">Benchmarking spatial and single-cell transcriptomics integration methods for transcript distribution prediction and cell type deconvolution</a></li>
</ul>
<h4 id="selected-tools">Selected tools</h4>
<ul>
<li>PASTE: <a href="https://www.nature.com/articles/s41592-022-01459-6">Alignment and integration of spatial transcriptomics data from multiple adjacent tissue slices</a> (<a href="https://github.com/raphael-group/paste">Python</a>)</li>
<li>GLUE: <a href="https://www.nature.com/articles/s41587-022-01284-4">Multi-omics single-cell data integration and regulatory inference with graph-linked embedding</a> (<a href="https://github.com/gao-lab/GLUE">Python/R</a>)</li>
<li>CellTrek: <a href="https://www.nature.com/articles/s41587-022-01233-1">Spatial charting of single-cell transcriptomes in tissues</a> (<a href="https://github.com/navinlabcode/CellTrek">R</a>)</li>
<li>Tangram: <a href="https://www.nature.com/articles/s41592-021-01264-7">aligns sc/snRNA-seq data to spatial data</a> (<a href="https://github.com/broadinstitute/Tangram">Python</a>)</li>
<li>SPOTlight: <a href="https://academic.oup.com/nar/article/49/9/e50/6129341">integration of Spatial Transcriptomics with scRNA-seq data to infer the location of cell types and states</a> (<a href="https://github.com/MarcElosua/SPOTlight">R</a>)</li>
<li>Harmony: <a href="https://www.nature.com/articles/s41592-019-0619-0">Fast, sensitive and accurate integration of single-cell data</a> (<a href="https://github.com/immunogenomics/harmony">R</a>)</li>
<li><a href="https://cell2location.readthedocs.io/en/latest/">cell2location</a>: <a href="https://www.nature.com/articles/s41587-021-01139-4">Cell2location maps fine-grained cell types in spatial transcriptomics</a> (<a href="https://github.com/BayraktarLab/cell2location/">Python</a>)</li>
</ul>
<hr>
<h3 id="single-cell-data-interactive-visualisation">Single cell data interactive visualisation</h3>
<ul>
<li><a href="https://academic.oup.com/nargab/article/2/3/lqaa052/5877814">Comparison of visualization tools for single-cell RNAseq data</a></li>
<li><a href="https://satijalab.org/seurat/articles/visualization_vignette.html#interactive-plotting-features-1">Seurat interactive plotting</a> (<code>HoverLocator()</code>)</li>
</ul>
<h3 id="automated-cell-type-annotation-turorialhttpswwwnaturecomarticless41596-021-00534-0">Automated cell type annotation <a href="https://www.nature.com/articles/s41596-021-00534-0">Turorial</a></h3>
<h4 id="reference-based">Reference-based</h4>
<ul>
<li><a href="https://github.com/dmcable/spacexr">RCTD/spacexr</a> (R)</li>
<li>scmap</li>
<li><a href="https://bioconductor.org/packages/devel/bioc/vignettes/SingleR/inst/doc/SingleR.html">SingleR</a></li>
<li><a href="https://www.nature.com/articles/s41587-021-00896-6">CelliD: clustering-free extraction of per-cell gene signatures</a> (<a href="https://github.com/RausellLab/CelliD">R</a>)</li>
</ul>
<h4 id="marker-based">Marker-based</h4>
<ul>
<li><a href="https://www.bioconductor.org/packages/devel/bioc/vignettes/AUCell/inst/doc/AUCell.html">AUCell: cells with active gene sets</a></li>
<li>GSEA/GSVA (for clusters)</li>
<li><a href="https://github.com/jcao89757/SCINA">SCINA</a> (for cells)</li>
</ul>
<h4 id="knowledge-based-clustering-and-annotation">Knowledge-based clustering and annotation</h4>
<ul>
<li><a href="https://genome.cshlp.org/content/early/2022/06/28/gr.276609.122.long">UNIFAN: using known gene sets + neural network</a></li>
</ul>
<hr>
<h3 id="other-tools">Other tools</h3>
<ul>
<li>sceasy: easy conversion of different single-cell data formats (<a href="https://github.com/cellgeni/sceasy">R</a>)</li>
</ul>

</content>
<p>
  
</p>

  </main>
  <footer>Made with <a href="https://github.com/janraasch/hugo-bearblog/">Hugo ʕ•ᴥ•ʔ Bear</a>
</footer>

    
</body>

</html>
